ShoppingCartApp/src/test/java/com/example/SpringApp/ServiceTest/ItemServiceTest.java

```diff
@@ -0,0 +1,58 @@
+package com.example.SpringApp.ServiceTest;
```

Comment:
What was the code coverage? Since this is a simple project, 100% code coverage is expected.